bundle/config/mutator/python/python_mutator_test.go

```diff
@@ -120,7 +126,7 @@ func TestPythonMutator_loadResources(t *testing.T) {
 
 		assert.Equal(t, []dyn.Location{
 			{
-				File:   "src/examples/job1.py",
+				File:   pathlib.Join(rootPath, "src/examples/job1.py"),
```

Comment:
I suspect you need to use `filepath` here as well, or it won't test equal on Windows.

Reply:
👍 , I keep forgetting which one is which